lighthouse-cli/test/smokehouse/frontends/smokehouse-bin.js

```diff
@@ -76,7 +79,7 @@ async function begin() {
     .alias({
       'jobs': 'j',
     })
-    .choices('runner', ['cli'])
+    .choices('runner', ['cli', 'bundle'])
```

Comment:
should there be a `--runner-path`, so others could make use a custom runner w/o changing this file?

makes sense to just wait for someone to request this feature, but it's easy enough to do perhaps we should just do it.

Reply:
Actually, _I_ will be one such user in the near future :)

Reply:
> should there be a `--runner-path`, so others could make use a custom runner w/o changing this file?

Could we just add it when you need it? Agreed it fits in easily, these are more or less just shorthand forms of that, but I'd like to avoid messing with yargs more right now :)

Reply:
> should there be a `--runner-path`, so others could make use a custom runner w/o changing this file?

though `smokehouse.js` *can* already take any custom runner without changes. I kind of figured that when we needed a really custom runner, we would also need firehouse's test filtering and so could just make a new frontend (`smoke-mask` :D). Like 80% of this file is parsing argv anyways.